### PR TITLE
Update documentation for new shipping locales automation

### PR DESCRIPTION
**Pull request by flodolo**
See https://github.com/mozilla-l10n/stores_l10n/pull/123

### CODE REVIEW

**Comment by flodolo** [comment]
@TheoChevalier Ping? 🦄

File: tools/stores_l10n/add_new_product.md [hunk 1]
```diff
@@ -6,12 +6,47 @@ This document will explain how to start tracking a new project, using Focus for
 
 ## Add new product to configuration
 
-Configuration for the Stores web app is in the `Project` class (`app\classes\Stores\Project.php`). The first thing to do is to add the project to `$products_data`, and you’ll need to determine beforehand:
+Configuration for the Stores web app is split between `app/config/product_sources.json` and the `Project` class (`app/classes/Stores/Project.php`). The first thing to do is to to determine:
```
Comment (TheoChevalier) [inline tools/stores_l10n/add_new_product.md]: to to

**Review by TheoChevalier — CHANGES_REQUESTED** [review]
Sorry, missed this one… (Also discovering GitHub rich diff, that’s great)
Just found a small repetition, otherwise looks good :)

**Comment by flodolo** [comment]
Updated, thanks (will need a green tick to make GH happy)

**Review by TheoChevalier — APPROVED** [review]

**Pull request merged**